src/layouts/main.html

```diff
@@ -0,0 +1,40 @@
+<!-- Places Section -->
+<div class="places_section">
+    <h1 class="places_section_title">Places</h1>
+    <div class="places_list_container">
+        <div class="places_list_a">
+            <include src="./components/places-list-a.html" locals='{
+            "classList":"places_list_a_item",
+            "list": ["SAN FRANCISCO", "SOUTH LILIANA", "BLICKVIEW", "NEW RAVEN"]
+          }'>
+            </include>
+        </div>
+        <div class="places_list_select">
+            <include src="./components/places-list-select.html" locals='{
+            "classList": "places_list_select_item",
+            "list": ["SAN FRANCISCO", "SOUTH LILIANA", "BLICKVIEW", "NEW RAVEN"]
+          }'>
+            </include>
+        </div>
+    </div>
+    <div class="img_container">
+        <div class="img">
+            <img src="../assets/images/desktop/place_img1.png" alt="Place IMG 1">
```

Comment:
Check the document I sent you (`how to work with Img`). And review all of your project!